subprojects/core/src/test/groovy/org/gradle/api/internal/file/BaseDirFileResolverSpec.groovy

```diff
@@ -128,10 +128,10 @@ class BaseDirFileResolverSpec extends Specification {
 
     def "normalizes file system roots"() {
         expect:
-        normalize(root) == root
+        normalize(root) == new File(root)
```

Comment:
The test was kinda useless before, because when you pass an absolute `File`, we just return it straight back again. Instead we now check the more interesting String parsing case.